ember-phone-input/src/services/phone-input.ts

```diff
@@ -0,0 +1,28 @@
+import Service from '@ember/service';
+import { resolve, all } from 'rsvp';
+import type intlTelInput from 'intl-tel-input';
+
+export default class PhoneInputService extends Service {
+  intlTelInput:
+    | ((
+        node: Element,
+        options?: intlTelInput.Options | undefined
+      ) => intlTelInput.Plugin)
+    | null = null;
```

Comment:
Do you think is it worth to have the type specified in an `interface` to keep this more readable?

Reply:
We have a lint rule that prevents us from using aliases which means we actually shouldn't create custom types just to make code cleaner